### ISSUE TITLE
Issues parsing Tisseo (Toulouse, France) GTFS data

### ISSUE THREAD

**Issue by Kernald** [jarondl/pygtfs#23]
I have an issue parsing [Tisseo GTFS files](https://data.toulouse-metropole.fr/explore/dataset/tisseo-gtfs/table/). Using [home-assistant](https://github.com/balloob/home-assistant), here's the error I have:

```
Traceback (most recent call last):
  File "/home/kernald/home-assistant/homeassistant/helpers/entity_component.py", line 91, in _setup_platform
    discovery_info)
  File "/home/kernald/home-assistant/homeassistant/components/sensor/gtfs.py", line 157, in setup_platform
    config["origin"], config["destination"]))
  File "/home/kernald/home-assistant/homeassistant/components/sensor/gtfs.py", line 176, in __init__
    self.update()
  File "/home/kernald/home-assistant/homeassistant/components/sensor/gtfs.py", line 215, in update
    self._data_source))
  File "/home/kernald/.homeassistant/lib/pygtfs/loader.py", line 87, in append_feed
    instance = gtfs_class(feed_id=feed_id, **records_as_dict)
  File "<string>", line 4, in __init__
  File "/home/kernald/.homeassistant/lib/sqlalchemy/orm/state.py", line 306, in _initialize_instance
    manager.dispatch.init_failure(self, args, kwargs)
  File "/home/kernald/.homeassistant/lib/sqlalchemy/util/langhelpers.py", line 60, in __exit__
    compat.reraise(exc_type, exc_value, exc_tb)
  File "/home/kernald/.homeassistant/lib/sqlalchemy/util/compat.py", line 184, in reraise
    raise value
  File "/home/kernald/.homeassistant/lib/sqlalchemy/orm/state.py", line 303, in _initialize_instance
    return manager.original_init(*mixed[1:], **kwargs)
  File "/home/kernald/.homeassistant/lib/sqlalchemy/ext/declarative/base.py", line 649, in _declarative_constructor
    setattr(self, k, kwargs[k])
  File "/home/kernald/.homeassistant/lib/sqlalchemy/orm/attributes.py", line 224, in __set__
    instance_dict(instance), value, None)
  File "/home/kernald/.homeassistant/lib/sqlalchemy/orm/attributes.py", line 701, in set
    value, old, initiator)
  File "/home/kernald/.homeassistant/lib/sqlalchemy/orm/attributes.py", line 710, in fire_replace_event
    self._init_append_or_replace_token())
  File "/home/kernald/.homeassistant/lib/sqlalchemy/orm/util.py", line 118, in set_
    return validator(state.obj(), key, value)
  File "/home/kernald/.homeassistant/lib/pygtfs/gtfs_entities.py", line 57, in in_range
    raise ValueError("Empty value not allowed in {0}".format(key))
ValueError: Empty value not allowed in wheelchair_boarding
```

Relevant home-assistant ticket is here: balloob/home-assistant#1761

**Comment by jarondl** [comment]
Hi @kernald,
This is a bug. (with an easy fix)
The 'parent' stations, starting with the red-cross school, have empty `wheelchair_boarding` fields, but this is OK according to the specs. We just need to fix the validations in `gtfs_entities`.
I will be able to give this attention in a few days.

Thank you for letting us know,
Yaron

    stop_id,stop_code,stop_name,stop_lat,stop_lon,location_type,parent_station,wheelchair_boarding
    ...
    3377704015496566,23055,Oncopole,43.5543,1.43191,0,1970324839361574,1
    3377704015496569,16937,Castelmaurou Mairie,43.6784,1.53146,0,1970324837185193,1
    1970324837184593,,Ecole Croix-Rouge,43.6052,1.38705,1,,
    1970324837184594,,Marengo-SNCF,43.6104,1.45524,1,,
    ...

**Comment by Kernald** [comment]
Great, thank you!

**Comment by jarondl** [comment]
Fixed in 0.1.3
